python/sparkdl/utils.py

```diff
@@ -20,38 +21,64 @@
 """
 import logging
 
-from pyspark import SparkContext
+from pyspark import SparkContext, SQLContext
 from pyspark.sql.column import Column
 
-ENTRYPOINT_CLASSNAME = "com.databricks.sparkdl.python.PythonInterface"
-
 logger = logging.getLogger('sparkdl')
 
-def _java_api_sql(javaClassName, sqlCtx = None):
-    """
-    Loads the PythonInterface object (lazily, because the spark context needs to be initialized
-    first).
+class JVMAPI(object):
```

Comment:
Sometimes it can be useful to create a namespace, but in this case I'd actually move it into it's own module. It should be cleaner that way. For example within the module you can drop the cls in statements like `cls._curr_sc()._jvm` and use `_curr_sc()._jvm`. You'll also get the benefits of a namespace, if you call the file `jvmapi.py` you'll use `jvmapi.for_class` and so on.